script.js

```diff
@@ -35,47 +35,128 @@ function getHumanChoice() {
 // Play 5 rounds by calling playRound 5 times.
 
 //     Hint: When you assign a function call to a variable, the return value of that function is assigned to the variable. Accessing the variable afterward will only provide the assigned value; it doesn’t recall the function. You need to recall the choice functions to get new choices for each round.
-
-function playGame() {
-  // Declare score values
-  let humanScore = 0;
-  let computerScore = 0;
-  // Loop function, run it 5 times
-  for (let step = 0; step <5; step++) {
-  // Function that has the main game logic
+let humanScore = 0;
+let computerScore = 0;
   function playRound(humanChoice, computerChoice) {
     if (
       (humanChoice == "rock" && computerChoice == "paper") ||
-      (humanChoice == "paper" && computerChoice == "scissor") || 
+      (humanChoice == "paper" && computerChoice == "scissor") ||
       (humanChoice == "scissor" && computerChoice == "rock")
     ) {
       console.log(`You lose! ${computerChoice} beats ${humanChoice}`);
+      gameData.classList.add("gameData");
+      gameData.textContent = `You lose! ${computerChoice} beats ${humanChoice}`;
```

Comment:
Consider putting this message in a variable:

```suggestion
      const msg = `You lose! ${computerChoice} beats ${humanChoice}`;
      console.log(msg);
      gameData.classList.add("gameData");
      gameData.textContent = msg;
```

Reply:
And same for the other messages